configs/generic_x86-64_hdd_config

```diff
@@ -39,6 +39,9 @@ MENDER_STORAGE_TOTAL_SIZE_MB=16000
 # Nothing to copy
 MENDER_COPY_BOOT_GAP="n"
 
+# Not using UBoot.
+MENDER_GRUB_EFI_INTEGRATION="y"
+
```

Comment:
This is the default value, as specified in,

https://github.com/mendersoftware/mender-convert/blob/7b6d957170236f139e5022d0d7d74a726ad6bae3/configs/mender_convert_config#L175

Do not see a huge benefit of being explicit about it here.

Reply:
Dang. My wrapper script didn't include that config file.  Agreed this is unneeded.